purr/patterns/right-sidebar.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Title: Right Sidebar
+ * Slug: purr/right-sidebar
+ * Categories: sidebar
+ */
```

Comment:
I wonder if  `Block Types: core/template-part/sidebar` would work? I don't know if we have a replace flow for it or not